src/catalogs/aviris/main.py

```diff
@@ -2,9 +2,15 @@
 
 import pystac
 import stacframes
+import logging
+import os
 
 from aviris_df import AvirisClassic, AvirisNg, AVIRIS_DESCRIPTION
 
+# set a configurable logging level for the entire app
+LOG_LEVEL = os.environ.get('LOG_LEVEL', 'INFO').upper()
+logging.basicConfig(format='[%(asctime)s|%(levelname)s|%(name)s|%(lineno)d] %(message)s', level=LOG_LEVEL)
```

Comment:
@rbreslow if that is too verbose we can always change it to smth more nice

Reply:
Indeed, I wanted the time since the programm started. I replaced it with `$(relativeCreated)d`, it's in ms and not exactly what i want but requires no extra code.